test/support/index.js

```diff
@@ -19,3 +19,5 @@ import './commands';
 import './coverage';
 
 import './api/patient';
+import './api/actions';
```

Comment:
can we alphabetize these?  because why not?

Reply:
whoops, saw that and thought of it and then didn't remember to do it